gherkin/CHANGELOG.md

```diff
@@ -10,10 +10,32 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 ## [Unreleased] - In Git
 
 ### Added
+<<<<<<< HEAD
```

Comment:
Hey @aslakhellesoy, are you aware of the conflict here? :)

Reply:
Well spotted! Will fix.

Reply:
There's also another further down, in the Perl changelog.